fdbserver/include/fdbserver/IPageEncryptionKeyProvider.actor.h

```diff
@@ -356,8 +395,16 @@ class AESEncryptionKeyProvider : public IPageEncryptionKeyProvider {
 
 	int64_t getEncryptionDomainIdFromHeader(const void* encodingHeader) override {
```

Comment:
This function seems to be a duplicate of `IPageEncryptionKeyProvider:: getEncryptionDomainIdFromHeader`?

Reply:
one is for `RandomEncryptionKeyProvider` and one is for `AESEncryptionKeyProvider`.

Reply:
Any way to make it a helper function?